### ISSUE TITLE
fix: interstitial on iOS not working

### ISSUE THREAD

**Issue by thekorn** [thekorn/xandr#61]

**Comment by Thumbler83** [comment]
Possible solution: reference the parent view controller instead of "self" in the display method.

At the moment, in the 'showInterstitialAd' method there are two calls to 'interstitialAd?.display' with the "from" parameter set to "self". Switching from "self" to the parent view controller I've been able to see the Ad. Here's what I changed in the code:

1) Create a method to fetch the parent view controller
```
private func parentController() -> UIViewController {
    var topController: UIViewController = UIApplication.shared.keyWindow!.rootViewController!
        while (topController.presentedViewController != nil) {
            topController = topController.presentedViewController!
        }
        return topController
    }
```

2) Change the from parameter in the "display" method

```
self.interstitialAd?.display(from: self.parentViewController())
self.interstitialAd?.display(from: self.parentViewController(), autoDismissDelay: Double(autoDismissDelay!))
```

**Comment by thekorn** [comment]
this is fixed with current release (0.2.13) - closing

https://github.com/user-attachments/assets/e91e3eca-cd1e-4336-a520-e7488e6f04bb
